getting_hired/preparing_for_job_search/project_portfolio.md

```diff
@@ -13,7 +13,6 @@ Don't underestimate the importance of high quality documentation on your project
 Remember to write some documentation for both technical and non-technical audience.  A potential employer should be able to follow your documentation, regardless of their technical background.
 
 * [A Beginner’s Guide to Writing Documentation](http://www.writethedocs.org/guide/writing/beginners-guide-to-docs/)
```

Comment:
It looks like removing the previous information also removed a empty line, which needs to be put back.